### PR TITLE
docs(): v6

### PR DESCRIPTION
**Pull request by ShaMan123**
## Motivation

beta

## Description

docs to support beta

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,10 +80,16 @@ $ yarn add fabric
 ```
 
 ```js
-// es6 imports
+// v6 es
+import * as fabric from 'fabric'; // browser
```
Comment (asturur) [inline README.md:84]: Let's also show the `import { Canvas, Rect } from 'fabric'`

Comment (ShaMan123) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -69,10 +80,16 @@ $ yarn add fabric
...
+import * as fabric from 'fabric/node'; // node
+// v6 cjs
+const fabric = require('fabric'); // browser
+const fabric = require('fabric/node'); // node
```
Comment (asturur) [inline README.md:88]: so node14 supports import and so do most of the bundlers.
What is the reason we want to keep require in the documentation?

**Review by asturur — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ShaMan123) [reply README.md:88]: I don't mind removing it

**Comment by ShaMan123** [comment]
go ahead

File: README.md (hunk 2 above)
Comment (ShaMan123) [reply README.md]: done

**Review by asturur — APPROVED** [review]

**Pull request merged**